src/elements/content-sidebar/versions/VersionsSidebar.scss

```diff
@@ -37,3 +37,8 @@
         padding-bottom: 20px;
     }
 }
+
+.btn-plain.bcs-Versions-upgrade {
```

Comment:
Is the `.btn-plain` portion of this needed? We typically see the something like the following to override the default plain button styling:

```
.bcs-Versions-upgrade {
    &,
    &:hover,
    &:active {
        color: $bdl-box-blue;
        font-weight: bold;
    }
}

Reply:
So the default if I change nothing is no bold ever. If I have what you suggest there I have blue text, and it is bold only when I hover? 😭  Looking into it hahaha

Reply:
I'm surprised if it works without that. `font-weight: normal` is specified for the hover state.

https://github.com/box/box-ui-elements/blob/master/src/styles/common/_buttons.scss#L137-L153